readme.txt

```diff
@@ -1,64 +1,91 @@
+= Get Started = 
+Installation is free, quick, and easy. Set up <a href="https://jetpack.com/install?from=wporg" rel="nofollow">the free plan</a> in minutes.
 
 == Installation ==
 
-[Install Jetpack from our site](https://jetpack.com/install?from=wporg) by typing in your site address.
+= Automated Installation =
+Installation is free, quick, and easy. [Install Jetpack from our site](https://jetpack.com/install?from=wporg) in minutes.
 
-Alternatively install Jetpack via the plugin directory, or by uploading the files manually to your server. After activating Jetpack, connect to WordPress.com to enable the Jetpack features.
+= Manual Alternatives =
+Alternatively install Jetpack via the plugin directory, or by uploading the files manually to your server, and then follow the on-screen instructions. If you need additional help [read our detailed instructions](http://jetpack.com/support/installing-jetpack/).
 
-If you need additional help [read our detailed instructions (including a video walkthrough)](http://jetpack.com/support/installing-jetpack/).
+== Frequently Asked Questions ==
 
+= Is Jetpack Free? =
+Yes! Jetpack's core features are and always will be free. 
 
-== Frequently Asked Questions ==
+These include: [site stats](https://jetpack.com/features/traffic/site-stats), a [high-speed CDN](https://jetpack.com/features/writing/content-delivery-network/) for images, [related posts](https://jetpack.com/features/traffic/related-posts) indexing, [downtime monitoring](https://jetpack.com/features/security/downtime-monitoring), brute force [attack protection](https://jetpack.com/features/security/brute-force-attack-protection), [automated publishing](https://jetpack.com/features/traffic/automatic-publishing/), [sidebar customization](https://jetpack.com/features/writing/sidebar-customization/), and many more.
 
-= Is Jetpack free? =
+= Should I purchase a paid plan? =
+Jetpack's paid services include automated backups, security scanning, spam filtering, video hosting, site monetization, advanced SEO tools, and dedicated priority support.
 
-Yes! The core features of Jetpack are and always will be free. Jetpack also integrates with paid services like Akismet (anti-spam), VaultPress (backups), and VideoPress (video hosting) but these are not required for Jetpack to function.
+Not every site requires an extra layer of protection and advanced tools but if your site is important to you, you should [consider upgrading](https://jetpack.com/pricing).
 
 = Why do I need a WordPress.com account? =
 
-Many of our core features (like Photon, Stats, and Protect) make use of the WordPress.com cloud. For this to happen Jetpack requires a (free) WordPress.com account. If you don't have one already you can easily create one during Jetpack's connection process.
+Jetpack and its services are provided by WordPress.com so for Jetpack to function, a WordPress.com account is required. (Think of it like a Jetpack account!)

```

Comment:
Should we add the word "free" back in that sentence, to make it clear it doesn't cost anything to create an account on WordPress.com?